app/assets/javascripts/modules/help.js.coffee

```diff
@@ -0,0 +1,248 @@
+                manually set spatial boundaries (<i class="fa fa-fw fa-crop"></i>),
+                use the buttons to the right of the search box.</p>
+                <p>To start your search session over, click the eraser icon (<i class="fa fa-eraser"></i>) to clear all of your set filters.</p>
+                '
+      element: '#keywords'
+      placement: 'bottom'
+      showNext: true
+      },{
+      title: "Search Results"
+      content: '<p>Search results will be shown in the Matching Collections panel below.  Each result will have summary 
+                information along with relevant badges to allow you to quickly scan your search results to find the
+                right collection for you.  The panel can be resized by clicking and dragging the bar above the "Matching
+                Collections" tab.</p>
+                <p>To view more information about a collection, click on the <i class="fa fa-info-circle"></i> icon.</p>
+                <p>To view granules available for download, click anywhere on a collection.</p>
+                <p>Click on the <i class="fa fa-plus"></i> icon to add a collection to a project, which allows you to compare multiple collections.</p>
+'
+      element: '#collection-results-list'
+      placement: 'top'
+      showNext: true
+      },{
+      title: "Facets"
+      content: "<p>Refine your search further with available facets, such as:
+                <div style='margin-left: 15px;'><ul style='list-style-type: disc;'>
+                  <li>Features</li>
+                  <li>Keywords</li>
+                  <li>Platforms</li>
+                  <li>Instruments</li>
+                  <li>Organizations</li>
+                  <li>Projects</li>
+                  <li>Processing Levels</li>
+                </ul></div></p>"
+      element: "#master-overlay-parent"
+      placement: 'right'
+      showNext: true
+      top: null
+    }, {
+      title: "Map Tools"
+      content: '<p>Use these standard map tools to configure and position the map as well as enable certain spatial search tools.</p>'
+      placement: 'left'
+      element: '.leaflet-control-zoom-in'
+      showNext: true
+    }, {
+      title: 'Toolbar'
+      content: '<p>Use the options available (upon logging in) in the application toolbar to view recent downloads, saved projects, and profile 
+      information. You can provide feedback using our feedback module.</p>'
+      element: '.user-info'
+      placement: 'bottom'
+    }]
+
+  defaultHelpOptions =
+    placement: 'auto left'
+    html: true
+    wait: true
+    trigger: 'manual'
+    template: defaultTemplate
+    container: 'body'
+
+  defaultTourOptions =
+    reflex: true
+    placement: 'left'
+
+  queue = []
+  shown = {}
+  index = 0
+  next = null
+  close = null
+
+  tourRunning = false
+
+  doNotShowTourAgain = false
+
+  toggleHideTour = ->
+    doNotShowTourAgain = if doNotShowTourAgain then false else true
```

Comment:
@mightynimble - hmmmmm If I remove line 112 above, then the 'End Tour' button no longer works.  It's needed.  :\